src/Portable.Xaml/Portable.Xaml/XamlXmlReader.cs

```diff
@@ -171,7 +171,7 @@ static XmlReaderSettings CreateReaderSettings(XamlXmlReaderSettings settings, Co
 				CloseInput = closeInput ?? settings?.CloseInput ?? false,
 				IgnoreComments = true,
 				IgnoreProcessingInstructions = true,
-				IgnoreWhitespace = true,
+				//IgnoreWhitespace = true,
```

Comment:
Is this test code?  or is IgnoreWhitespace's default false?

Reply:
I think this is not a test code. When ```IgnoreWhitespace = false``` then inline tests go fault. Can I just remove commented line in this case?

Reply:
Yes that would be perfect.